android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/req/ReqInitialize.kt

```diff
@@ -20,6 +20,7 @@ import com.getcapacitor.JSObject
 class ReqInitialize(data: JSObject) {
     val licenseKey: String
     val productKey: String
+    val googleId: String
```

Comment:
it should be nullable. If the user doesn't want to add google sign in leaves it empty.